queries.sql

```diff
@@ -9,4 +9,59 @@ SELECT * from animals WHERE neutered=TRUE;
 SELECT * from animals WHERE name <> 'Gabumon';
 SELECT * from animals WHERE weight_kg >=10.4 and weight_kg <= 17.3;
 
+-- day2 questions
+ALTER TABLE animals ADD species VARCHAR(50);
```

Comment:
You altered the table correctly but be sure to add this line in your schema file because anything that creates or changes the table structure should be saved there 📝

Reply:
Sure, I am working on it now and I will push the changes soon